src/services/item/ws/hooks.ts

```diff
@@ -94,9 +96,9 @@ function registerMemberItemsTopic(websockets: WebsocketService, itemService: Ite
   // on create, notify own items of creator with new item IF path is root
   itemService.hooks.setPostHook('create', async (actor, repositories, { item }) => {
     const parentId = getParentFromPath(item.path);
-    if (parentId === undefined) {
+    if (parentId === undefined && actor) {
       // root item, notify creator
-      websockets.publish(memberItemsTopic, item.creator.id, OwnItemsEvent('create', item));
+      websockets.publish(memberItemsTopic, actor.id, OwnItemsEvent('create', item));
```

Comment:
Is `actor` always the creator here? So shouldn't it simply check `if (parentId === undefined && item.creator)`?